package.json

```diff
@@ -80,11 +79,11 @@
     "react": "^15.4.2",
     "react-addons-test-utils": "15.4.2",
     "react-helmet": "^3.3.2",
-    "react-intl": "^2.2.3",
-    "react-redux": "^5.0.2",
-    "react-router": "^3.0.2",
-    "redux": "^3.5.2",
-    "rxjs": "^5.0.3",
+    "react-intl": "2.2.3",
+    "react-redux": "5.0.2",
+    "react-router-dom": "^4.0.0-beta.6",
```

Comment:
i've noticed `yarn add <package>` adds the `^` notation to versions, here and on a few other lines. Just double checking that we're intentionally adding it, and not just missing it.